### DISCUSSION TITLE
reactive potential by deepmd-kit

### DISCUSSION THREAD

**Discussion by zjd11** [deepmodeling/deepmd-kit#1287] [Q&A]
Hello, developer of deepmd-kit
I want to train a reactive potential by deepmd-kit. I have noticed a example input file in dpgen in which force are calculated by GAUSSIAN, NOT by VASP.
I want to know whether vasp can be used to calculate force and produce the train set for deepmd-kit.
Or in this case, what are the advantages of using Gaussian than VASP?
Thanks for your time.
jidong

**Comment by njzjz** [comment]
DeePMD-kit does not relate to any specific first-principle software.

You can use whatever you like.
